### PR TITLE
Arn 770 handle multiple records

### PR DESCRIPTION
**Pull request by wood-n**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/assessments/controller/AssessmentControllerTest.kt [hunk 1]
```diff
@@ -38,7 +38,7 @@ import java.util.UUID
     executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
   )
 )
-@AutoConfigureWebTestClient
+@AutoConfigureWebTestClient(timeout = "6000000")
```
Comment (markheyward-chaplin) [inline src/test/kotlin/uk/gov/justice/digital/assessments/controller/AssessmentControllerTest.kt:41]: Do we want this timeout checked in?

**Review by markheyward-chaplin — DISMISSED** [review]
LGTM. The only comment I'd make is to run this by @SWhitlam to ensure he's happy with structured answers being stored as json.

File: src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ server:
   forward-headers-strategy: native
 spring:
   flyway:
-    enabled: true
+    enabled: false
```
Comment (SWhitlam) [inline src/main/resources/application-dev.yml:9]: Did you intend to check this in?

Comment (wood-n) [reply src/main/resources/application-dev.yml:9]: Nope! Thanks for catching

**Review by SWhitlam — APPROVED** [review]

**Pull request merged**